tools/device-listing/device-listing.cs

```diff
@@ -210,13 +214,11 @@ private static string GetRelativePathSimple(string path, string parentPath)
         {
             if (path.StartsWith(parentPath))
             {
-                int i = parentPath.Length;
-                if (path[i] == '/' || path[i] == '\\')
-                {
-                    i++;
-                }
+                string fileName = Path.GetFileName(path);
+                string dirName = new DirectoryInfo(path).Parent.Name;                    
+                UriBuilder uriBuilder = new UriBuilder(){Path = Path.Combine(dirName, fileName)};
```

Comment:
I'd leave this logic as it was since it worked correctly according to the name but instead change the logic for generating links (inside of `GetDeviceListing`) since that is actually incorrect (I'd just to `Replace("\\", "/")` but Uri is fine too - you might want to add utility like `CreateMarkdownLinkFromPath` or similar)

Reply:
also nit: space after and before `{` and `}`

Reply:
I thought about using `Replace("\\", "/")`, but I needed to add logic to know which operating system is running.